corona-warn-app/cwa-app-android
# PPA should only collect data for PCA tests (EXPOSUREAPP-6598)

### Description
This PR makes sure that only PPA for PCA tests and not RAT tests are collected. PPA for RAT tests will be implemented in an upcoming sprint. In order for this to work, I had to pass the `testType` to a few additional fragments via `navArgs`.

### How to test: 
- Submit positive PCR test
- Check TestMenu => Data Donation => Create Current Analytics (card on the very bottom)
- values for `key_submission_metadata_set` and `test_result_metadata_set ` should be there

- reset app
- Submit positive RAT test 
- Check TestMenu => Data Donation => Create Current Analytics (card on the very bottom)
- **No** values for `key_submission_metadata_set` and `test_result_metadata_set ` should be there
